Resources/config/security_expressions.xml

```diff
@@ -27,7 +27,7 @@
 
     <services>
         <service id="security.extra.twig_extension" class="%security.extra.twig_extension.class%" public="false">
-            <argument type="service" id="security.context" />
+            <argument type="service" id="security.authorization_checker" />
```

Comment:
this means you are dropping support for Symfony < 2.6. you should either change the composer.json to make it part of the requirements, or keep support for older versions in parallel (which is possible and may be better as 2.3 is still a maintained version)